hoomd/mpcd/Sorter.cc

```diff
@@ -43,7 +43,7 @@ void mpcd::Sorter::update(uint64_t timestep)
 
     // generate and apply the sorted order
     computeOrder(timestep);
-    applyOrder();
+    applySortOrder();
```

Comment:
Why was this renamed? It's confusing to have `computeOrder` but `applySortOrder`. I would revert to `applyOrder`

Reply:
It was done to make it match `SFCPackTuner`, which has a function `applySortOrder`. It was just to standardize the names of tuner functions. `computeOrder`'s equivalent in `SFCPackTuner` is `getSortedOrder2D()` and `getSortedOrder3D()`. We could standardize it more by renaming `computeOrder` to `getSortedOrder` or revert back to `applyOrder`.

Reply:
Please revert to the old name to minimize the number of nonessential changes.